cross/sonarr/Makefile

```
@@ -1,9 +1,9 @@
-PKG_NAME = NzbDrone
-PKG_VERS = 2.0.0.5344
+PKG_NAME = Sonarr
+PKG_VERS = 3.0.5.1144
 PKG_EXT = tar.gz
-PKG_DIST_NAME = $(PKG_NAME).master.$(PKG_VERS).mono.$(PKG_EXT)
-PKG_DIST_SITE = https://download.sonarr.tv/v2/master/mono
-PKG_DIR = NzbDrone
+PKG_DIST_NAME = $(PKG_NAME).main.$(PKG_VERS).linux.$(PKG_EXT)
+PKG_DIST_SITE = https://download.sonarr.tv/v3/main/$(PKG_VERS)
+PKG_DIR = Sonarr
```

Yes, I like the long lasted rename to Sonarr!
```suggestion
PKG_DIR = $(PKG_NAME)
```